### PR TITLE
HOTFIX 6217 any relationships empty content hotfix

### LINKED ISSUE #6217: Entity remains in relationship fields after being deleted
**Describe the bug**
When an entity related to others gets deleted the values in those other entities keep the reference to the deleted one. This value is present in the edition of the entity with the inconsistency, if the Save button is clicked the action doesn't get completed and the buttons remain disabled. Bulk edition actions like the edition of the template fix the problem.

**To Reproduce**
1. Create an entity with any template <A>
2. Create a template <T> with a Relationship field <R>
3. Create an entity with the template T with the field R pointing to A
4. Delete entity A
5. View entity B and see the value of R, A is still in field R

**Expected behavior**
All the references to A should be removed with its deletion.

**Screenshots**

_Before deleting entity_
Screenshot 2023-10-20 095505
_After deleting entity A_
Screenshot 2023-10-20 095617

### PR DESCRIPTION
**Pull request by fnocetti**
fixes #6217 

PR checklist:
- [ ] Update READ.me ?
- [ ] Update API documentation ?

QA checklist:
- [ ] Smoke test the functionality described in the issue
- [ ] Test for side effects
- [ ] UI responsiveness
- [ ] Cross browser testing
- [ ] Code review

### CODE REVIEW

File: app/api/entities/specs/fixtures.js [hunk 1]
```diff
@@ -674,7 +674,7 @@ export default {
           _id: db.id(),
           type: 'relationship',
           name: 'relationship_to_any_template',
-          content: '',
+          // No content set to test no content apart from content: ""
```
Comment (LaszloKecskes) [inline app/api/entities/specs/fixtures.js:677]: Is the `content: ''` case still tested somewhere?

**Review by LaszloKecskes — CHANGES_REQUESTED** [review]
Confirmed working as intended locally, by manually removing content from a template to test the situation described in the issue.

Code looks good, one question in a test.

File: app/api/entities/specs/fixtures.js (hunk 1 above)
Comment (fnocetti) [reply app/api/entities/specs/fixtures.js:677]: Yes, there's a case here: https://github.com/huridocs/uwazi/pull/6789/files#diff-7d284794be370388a325e4eaeb22025f7d0d71d26541459d7d287af920abb546L586

Comment (LaszloKecskes) [reply app/api/entities/specs/fixtures.js:677]: Thanks!

**Review by LaszloKecskes — APPROVED** [review]

**Comment by fnocetti** [comment]
I'm merging as we agreed the failing test is unrelated.

**Pull request merged**